kernel/.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Why is this file added? AFAIK we are configuring prettier through `package.json`: https://github.com/decentraland/explorer/blob/master/kernel/package.json#L10

Is this better somehow?

Reply:
I had problems with VSCode while formatting and opening the explorer folder (it works well opening explorer/kernel tho). I know @BrianAmadori also had a similar issue.